TODO/tensorflow-in-a-nutshell-part-one-basics.md

```diff
@@ -1,21 +1,21 @@
 > * 原文地址：[TensorFlow in a Nutshell — Part One: Basics](https://medium.com/@camrongodbout/tensorflow-in-a-nutshell-part-one-basics-3f4403709c9d#.y95bdu5wy)
 * 原文作者：[Camron Godbout](https://medium.com/@camrongodbout)
 * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者：
+* 译者：[cdpath](https://github.com/cdpath)
 * 校对者：
 
-# TensorFlow in a Nutshell — Part One: Basics
+# 简明 TensorFlow 教程 —— 第一部分：基础知识
 
 
 
 
-#### The fast and easy guide to the most popular Deep Learning framework in the world.
+#### 快速上手世界上最流行的深度学习框架
 
-TensorFlow is a framework created by Google for creating Deep Learning models. Deep Learning is a category of machine learning models that use multi-layer neural networks. The idea of deep learning has been around since 1943 when neurophysiologist Warren McCulloch and mathematician Walter Pitts wrote a paper on how neurons might work and they modeled a simple neural network using electrical circuits.
+TensorFlow 是谷歌开发的用于训练深度学习模型的框架。深度学习属于机器学习，使用多层神经网络。深度学习的思想早在 1943 年，神经生理学家 Warren McCulloch 和数学家 Walter Pitts 发表了关于神经元工作机制的论文之后就开始流行。他俩还用电路搭建了简单的神经网络模型。
```

Comment:
>The idea of deep learning has been around since 1943 when neurophysiologist Warren McCulloch and mathematician Walter Pitts wrote a paper on how neurons might work and they modeled a simple neural network using electrical circuits.

深度学习的概念是从1943年流行起来的，当时神经生理学家 Warren McCulloch 和数学家 Walter Pitts 发表了关于神经元工作机制的论文，他俩还用电路搭建了简单的神经网络模型。